2023/P_U2/main.cpp

```diff
@@ -0,0 +1,104 @@
+#include <fstream>
+#include <iostream>
+#include <vector>
+#include <iomanip>
+using namespace std;
+
+struct Mokinys
+{
+    string vardas;
+    vector<string> zymes;
+    int laimejimuKiekis = 0;
+    int lygiujuKiekis = 0;
+    int pralaimejimuKiekis = 0;
+};
+
+void skaitymas(int &mokiniuKiekis, Mokinys mokiniai[])
+{
+    ifstream data("U2.txt");
+    int zymiuKiekis;
+    char ch[14];
```

Comment:
Make it zero, to avoid potential buffer overflow when it is assigned to a string.